src/contracts/projects.ts

```diff
@@ -21,6 +21,11 @@ export type APIConfigEntity = {
   defaultValue: boolean | Record<string, any>;
   enabled: boolean;
   rules: Array<APIConfigRule>;
+  extraData: APIConfigSpecExtraData;
```

Comment:
this is going to get exposed to download_config_specs as well, but it shouldnt be.  Can you expose this just on the other endpoint so we dont bloat the download config specs response?